cmd/example-client/client.go

```diff
@@ -53,4 +53,14 @@ func main() {
 		p := pList.Items[i]
 		log.Printf("Index: %d Policy Targets: %+v\n", i, p.Spec.GetTargets())
 	}
+
+	// Test MeshPolicies
+	mpList, err := ic.AuthenticationV1alpha1().MeshPolicies().List(metav1.ListOptions{})
```

Comment:
Enhance this to test he following:
1) List all mesh policies and print out the names
2) For each named meshpolicy get the resource and print out some of its contents